json-guides/retry-timeout.json

```diff
@@ -28,7 +28,7 @@
             "name": "BankScenario",
             "title": "Example: bank scenario",
             "description": [
-                "Imagine that you're developing an online transaction history microservice for a bank to allow clients to check their transaction history. After a customer selects to view their transaction history, the initial naive implementation of the microservice loads indefinitely because the transaction history can not be retrieved from the back-end database that stores the transactions. As you progress through the guide, you will see how the transaction history request being handle using <code>@Timeout</code> and <code>@Retry</code> annotations."
+                "Imagine that you're developing an online transaction history microservice for a bank to allow clients to check their transaction history. After a customer selects to view their transaction history, the initial naive implementation of the microservice loads indefinitely because the transaction history can not be retrieved from the database that stores the transactions."
```

Comment:
We should reword the second sentence. The cause of the timeout is likely not because of our implementation, but because of some network, server, or database problem.